### PR TITLE
Add file CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by dataclouder**
New file CONTRIBUTING.md

File CODE_OF_CONDUCT.md was moved from .github to top level.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+### Updating pull requests
+
+If your PR fails to pass CI, or if you get change requests from the reviewers, you need to apply fixes in your local
+repository, and then submit the changes.
+
+``` shell
+# edit files
+git add filename
+git commit -v --signoff filename
```
Comment (Didainius) [inline CONTRIBUTING.md:89]: I'd guess we could just put another link to DCO where it explains what the signature must contain in `--signoff` as getting up to this place one will probably have already forgotten there was a link at the top.

**Review by Didainius — APPROVED** [review]
Left this only comment I have.

Thanks !

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
+# Contributing to terraform-provider-vcd
+
+The **terraform-provider-vcd** project team welcomes contributions from the community. 
+Before you start working with any contribution, please read our [Developer Certificate of Origin](https://cla.vmware.com/dco).
+All contributions to this repository must be signed as described on that page. Your signature certifies that you wrote the patch or have the right to pass it on as an open-source patch.
+
+
+## Community
+
+Terraform-provider-vcd contributors can discuss matters here:
```
Comment (lvirbalas) [inline CONTRIBUTING.md:10]:
```suggestion
Terraform VMware Cloud Director Provider contributors can discuss matters here:
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
+# Contributing to terraform-provider-vcd
```
Comment (lvirbalas) [inline CONTRIBUTING.md:1]:
```suggestion
# Contributing to Terraform VMware Cloud Provider
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+git checkout master
+git pull upstream master
+git push
+# At this point, your local copy of the master branch is synchronized
+git checkout my-new-feature
+git merge master
+```
+If there are conflicts you'll need to [merge them now](https://stackoverflow.com/questions/161813/how-to-resolve-merge-conflicts-in-git).
+
+### Updating pull requests
```
Comment (lvirbalas) [inline CONTRIBUTING.md:81]:
```suggestion
### Updating Pull Requests
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
+# Contributing to terraform-provider-vcd
+
+The **terraform-provider-vcd** project team welcomes contributions from the community. 
+Before you start working with any contribution, please read our [Developer Certificate of Origin](https://cla.vmware.com/dco).
+All contributions to this repository must be signed as described on that page. Your signature certifies that you wrote the patch or have the right to pass it on as an open-source patch.
+
+
```
Comment (lvirbalas) [inline CONTRIBUTING.md:7]: What about adding a short sentence how it is done here (git commit --signoff) and/or a link to "Updating Pull Requests" section for full details?

**Review by lvirbalas — CHANGES_REQUESTED** [review]
Looks great, just a few updates needed.

File: CONTRIBUTING.md (hunk 1 above)
Comment (dataclouder) [reply CONTRIBUTING.md:95]: done

File: CONTRIBUTING.md (hunk 2 above)
Comment (dataclouder) [reply CONTRIBUTING.md]: fixed

File: CONTRIBUTING.md (hunk 3 above)
Comment (dataclouder) [reply CONTRIBUTING.md:1]: changed

File: CONTRIBUTING.md (hunk 4 above)
Comment (dataclouder) [reply CONTRIBUTING.md]: changed

File: CONTRIBUTING.md (hunk 5 above)
Comment (dataclouder) [reply CONTRIBUTING.md:7]: done

**Review by lvirbalas — APPROVED** [review]

**Pull request merged**